packages/cli/src/handlers/dbt/run.ts

```diff
@@ -36,5 +37,9 @@ export const dbtRunHandler = async (
         });
         throw new ParseError(`Failed to run dbt:\n  ${e.message}`);
     }
-    await generateHandler({ ...options, assumeYes: true });
+    await generateHandler({
+        ...options,
+        assumeYes: true,
+        excludeDimensionTypes: !options.excludeDimensionTypes,
```

Comment:
Should we be inverting the boolean value here ?

Reply:
nope!!! good spot!